lib/milestone-db/index.js

```diff
@@ -0,0 +1,39 @@
+var emitter = require('../emitter');
+
+module.exports = MilestoneDB;
+function MilestoneDB(options) {
+  emitter.EventEmitter.call(this);
+
+  // The interval at which milestone snapshots should be saved
+  this.interval = options && options.interval;
+}
+emitter.mixin(MilestoneDB);
+
+MilestoneDB.prototype.close = function(callback) {
+  if (callback) callback();
```

Comment:
The base and memory DB implementations aren't super-consistent about ensuring their callbacks are async, but for these two new milestone classes, I think it'd be good to consistently `process.nextTick` before calling callbacks.

Reply:
Fine by me. I was unsure why I'd seen so many `nextTick`s around (I work in TypeScript and avoid callbacks wherever possible!). I had a read of [this blog post][1]. Is that pretty much the gist of what we're trying to achieve?

[1]: https://jsblog.insiderattack.net/event-loop-best-practices-nodejs-event-loop-part-5-e29b2b50bfe2

Reply:
> I had a read of this blog post. Is that pretty much the gist of what we're trying to achieve?

Yup, that's exactly it!